### PR TITLE
Make footer dynamic

### LINKED ISSUE #1850: Factor out footer information into config
Related to #1792, #1716

**Is your feature request related to a problem? Please describe.**
We have the footer values for igbo, yoruba & pidgin. We need to refactor the Footer container to not be hardcoded to the UK News values.

**Describe the solution you'd like**
Move the UK News values to config, get the values from config. Have some defaults for services which don't yet have values (ie. Persian)

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1850

**Overall change:** Make footer config dynamic

**Code changes:**

- Make footer config dynamic via service
- Make footer story
- Remove Thai due to need to keep it up to date pointlessly

Translations used can be found on WSFRONTPGS-24 and https://github.com/bbc/simorgh/issues/1716

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by dr3** [comment]
~BLOCKED ON https://github.com/bbc/psammead/issues/567~
~As Persian has too many items, causing a grid issue~

Now unblocked

**Comment by dr3** [comment]
Blocked on Igbo, Yoruba and Pidgin `Copyright` translations, as they all seem to be the same 🙈

**Review by j-pendlebury — APPROVED** [review]
Looks great 👍

**Review by DenisHdz — APPROVED** [review]

File: src/app/lib/config/services/persian.js [hunk 1]
```diff
@@ -102,6 +102,49 @@ const persian = {
...
+        href: 'https://www.bbc.com/persian/institutional-37542244',
+        text: 'تماس با بی بی سی',
+      },
+      {
+        href: 'https://www.bbc.co.uk/bbcnewsletter',
+        text: 'Get Personalised Newsletters',
+      },
+    ],
+    copyrightText:
+      'BBC. بی بی سی. بی بی سی مسئول محتوای سایت های دیگر نیست. سیاست ما درباره لینک دادن به سایت های دیگر.',
```
Comment (sareh) [inline src/app/lib/config/services/persian.js]:
```suggestion
      'بی بی سی. بی بی سی مسئول محتوای سایت های دیگر نیست. سیاست ما درباره لینک دادن به سایت های دیگر.',
```
The word 'BBC' is written here in English as well as in Farsi just following it `. Removing the duplicated text.

**Review by sareh — APPROVED** [review]
Looks great. Thanks for the fix, @dr3 
(Screenshot of http://localhost.bbc.co.uk:7080/persian/articles/c4vlle3q337o when manually adding `dir="rtl"` on the html tag) 🎉  
Screen Shot 2019-06-14 at 17 08 14

**Review by DenisHdz — APPROVED** [review]

**Comment by david-boydell** [comment]
Looks good, ready for merge.

**Pull request merged**